prime-router/src/main/kotlin/Report.kt

```diff
@@ -833,6 +835,38 @@ class Report : Logging {
         )
     }
 
+    /**
+     * Gets the item hash for a the [rowNum] of the report. Returns the ByteArray hash.
+     */
+    fun getItemHashForRow(rowNum: Int): String {
+        // calculate and store item hash for deduplication purposes for the generated item
+        val row = this.table.row(rowNum)
+        var rawStr = ""
+        for (colNum in 0 until row.columnCount()) {
+            rawStr += row.getString(colNum)
+        }
```

Comment:
Was there a specific reason you didn't want to use the `joinToString` method here?

Reply:
What do you mean here?  I have not found a way to get a TableSaw row to easily turn into one concatenated string - the columns are not necessarily String data types and this is not a list or array of strings.